src/figuresClasses.ts

```diff
@@ -1,19 +1,81 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
 
+type Color = 'red' | 'green' | 'blue';
+export interface Figure {
+  color: Color;
+  shape?: Shape;
```

Comment:
we have shape in every figure, I think it's neccesary parametr

Reply:
Agree, fixed